client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -578,6 +578,13 @@ $design-button-primary-color: rgb(17, 122, 201);
 					}
 				}
 			}
+
+		}
+
+		.design-picker-design-title__theme-tier-badge {
+			&:empty {
+				display: none;
+			}
```

Comment:
I'm using the `:empty` pseudo-selector because the `ThemeTierBadge` always returns a `div`, even if the badge is `null`. I thought it would be easier to hide this with styling than risking breaking designs elsewhere.